test/cell_multi_cell_refactor/cell.py

```diff
@@ -56,45 +53,10 @@ def test_ln_positioning_and_page_breaking_for_cell(self):
                 link="",
             )
 
-        outfile = relative_path_to("output.pdf")
-        doc.output(outfile)
-        known_good_hash = "d49bb2b7c9b3504ef45309a4a155cfc2"
-        self.assertEqual(calculate_hash_of_file(outfile), known_good_hash)
-        os.unlink(outfile)
-
-
-## Code used to create this test
-# doc = fpdf.FPDF(format = 'letter', unit = 'pt')
-# set_doc_date_0(doc)
-# doc.add_page()
-
-# my_text_size = 36
-# doc.set_font('Arial', size=my_text_size)
-# text = ('Lorem ipsum Ut nostrud irure reprehenderit anim nostrud dolore sed '
-#         'ut Excepteur dolore ut sunt irure consectetur tempor eu tempor '
```

Comment:
i thought this was useful?

Reply:
I can restore those comments if you want.
Given I did not follow those steps to generate the new reference PDF files, I thought I could remove this

Reply:
I think its a good idea to have the steps used to generate the files, (or at least some rough idea of what to do) because it removes some work for new tests to follow the same patterns and be uniform with the previous tests.

Reply:
ok